app/javascript/packages/document-capture/components/acuant-selfie-capture-canvas.scss

```diff
@@ -11,3 +11,7 @@
   padding: 10px;
   z-index: 11;
 }
+
+.document-capture-selfie-feedback:empty {
+  display: none;
```

Comment:
Since the feedback uses `aria-live`, I'd wonder of the compatibility of hiding content.

Related: https://www.davidmacd.com/blog/test-aria-live-display-none.html

Maybe we could approach this by only _adding_ `padding` and/or `background` in the styles above if `:not(:empty)` ?

Reply:
Oh interesting, thanks for that lin! I'm investigating now. Because the hint text in the document capture has no padding, removing the padding is another option that I think might work.

Reply:
I removed the padding. That fixes this and brings the styles closer to the other hint text (though I personally think it's better with padding).

Reply:
Do you think it's better to go the `:not(:empty)` route? I'm going to check with Kelli (designer) also.

Reply:
I think `:not(:empty)` could be an option if you want the padding.

Another idea is to apply the styles to an element inside the `aria-live` element, and conditionally render that in JavaScript.

e.g.

```tsx
<span aria-live="assertive">
  {feedback && <span class="document-capture-selfie-feedback">{feedback}</span>}
</span>
```

Reply:
That solution makes a lot of sense to me. I implemented that in favor of the more complex CSS selectors. Thanks!